src/test/java/dev/ocpd/jsensible/rules/spring/NoNullableMisuseInRepositories.java

```diff
@@ -5,23 +5,23 @@
 @SuppressWarnings("unused")
 public class NoNullableMisuseInRepositories {
 
-    public interface CompliantRepo extends JpaRepository<Object, Long> {
+    public interface Compliant extends JpaRepository<Object, Long> {
 
         @org.springframework.lang.Nullable
         Object findBy(String test);
 
         Object findObjectBy(@org.springframework.lang.Nullable String test);
     }
 
-    public interface NonCompliantRepo1 extends JpaRepository<Object, Long> {
+    public interface NonCompliant1 extends JpaRepository<Object, Long> {
 
         @jakarta.annotation.Nullable
         Object findBy(String test);
 
         Object findObjectBy(@jakarta.annotation.Nullable String test);
```

Comment:
I suggest we split this into two as we need both of them to be failing, if only one fails the test will count as a pass which is an issue.